pages/event-types/[type].tsx

```diff
@@ -722,6 +773,24 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                           </div>
                         </div>
                       </div>
+                      <div className="items-center block sm:flex">
```

Comment:
this should be wrapped into `{web3App && <Stuff />}` and only visible if it's enabled in app.cal.com/integrations
![image](https://user-images.githubusercontent.com/8019099/147754261-030474d9-aab7-4554-8ebf-7c51bd9a9f9a.png)

@edanfesi can you take care of this once we have `user_settings` in the db?

Reply:
yes, of course. I will do it